core/src/main/scala/org/apache/spark/rdd/RDDCheckpointData.scala

```diff
@@ -23,7 +23,10 @@ import org.apache.spark.Partition
 
 /**
  * Enumeration to manage state transitions of an RDD through checkpointing
- * [ Initialized --> checkpointing in progress --> checkpointed ].
+ *
+ * {{{
+ * [ Initialized --> checkpointing in progress --> checkpointed ]
+ * }}}
```

Comment:
- Java
  <img width="575" alt="2016-11-27 4 34 17" src="https://cloud.githubusercontent.com/assets/6477701/20642757/ceba8f0e-b45a-11e6-986b-024af7989f81.png">

Scaladoc not found